tools/qt-builder/README.md

```diff
@@ -104,16 +104,13 @@ Verify again
 1.  nss (needed for qtwebengine)  
 `sudo apt-get install libnss3-dev -y`  
 
-### Mac
-**TODO: Update this section for Qt 5.15.2**  
+### Mac  
 1.  git >= 1.6  
 Check if needed `git --version`  
 Install from https://git-scm.com/download/mac  
 Verify again  
-1.  pkg-config  
-brew fontconfig dbus-glib stall pkg-config  
-1.  dbus-1  
-brew install dbus-glib  
+1.  install pkg-config, dbug-glib, and fontconfig  
```

Comment:
You are using `1.` twice here.

Reply:
I followed same markup notation used in windows section, which renders correctly although in source looks to be 1. multiple times.  Please refer to original document source before my change.